lib/permit/rules.ex

```diff
@@ -20,9 +21,30 @@ defmodule Permit.Rules do
       |> apply(:list_groups, [])
       |> Enum.map(fn name ->
         quote do
-          @spec unquote(name)(Permit.t(), Types.resource(), Types.condition()) :: Permit.t()
-          def unquote(name)(authorization, resource, conditions \\ true) do
-            permission_to(authorization, unquote(name), resource, conditions)
+          # @spec unquote(name)(Permit.t(), Types.resource(), Types.condition()) :: Permit.t()
```

Comment:
I would like us to dig into this - it would be better if these generated functions did have typespecs. What error has made us comment this out now?